app/src/forms/submission/routes.js

```diff
@@ -2,34 +2,36 @@ const routes = require('express').Router();
 
 const controller = require('./controller');
 const currentUser = require('../auth/middleware/userAccess').currentUser;
+const hasSubmissionPermissions = require('../auth/middleware/userAccess').hasSubmissionPermissions;
```

Comment:
If we change the userAccess middleware file down the line to module.export an object with those functions as attributes, we can then do something nicer like the following:
```js
const { currentUser, hasSubmissionPermissions } = require('../auth/middleware/userAccess');
```